tests/test_metadata.py

```diff
@@ -8,86 +8,90 @@
 import pytest
 from requests_cache import CachedSession
 
-from research_index_backend.create_graph_from_doi import (
-    get_output_metadata,
-    score_name_similarity,
-)
+from research_index_backend.get_metadata import MetadataFetcher
+from research_index_backend.create_graph_from_doi import score_name_similarity
 
+@pytest.fixture
+def session():
+    return CachedSession()
 
-@pytest.mark.skip(reason="Requires access to OpenAire Graph API")
-def test_broken_doi():
-    """An incorrect DOI should raise an error"""
-    s = CachedSession()
+@pytest.fixture
+def fetcher(session):
+    return MetadataFetcher(session)
 
-    broken_doi = "10.1dd016/j.envsoft.2021"
-    with pytest.raises(ValueError) as ex:
-        get_output_metadata(s, broken_doi)
-    expected = "DOI 10.1dd016/j.envsoft.2021 returned no results"
-    assert str(ex.value) == expected
+class TestMetadataFetcher:
+    @pytest.mark.skip(reason="Requires access to OpenAire Graph API")
```

Comment:
This test is not needed, as invalid DOIs are now caught by the DOIManager.  What we should test for is when the OpenAire API returns a "403" as noted in #27